credentials/src/test/kotlin/web5/sdk/credentials/PresentationExchangeTest.kt

```diff
@@ -297,4 +301,102 @@ class PresentationExchangeTest {
       }.messageContains("Missing input descriptors: The presentation definition requires")
     }
   }
+
+  @Nested
+  inner class CreatePresentationFromCredentials {
+    @Test
+    fun `creates valid submission when VC satisfies tbdex PD`() {
+      val pd = jsonMapper.readValue(
+        readPd("src/test/resources/pd_sanctions.json"),
+        PresentationDefinitionV2::class.java
+      )
+
+      val presentationSubmission = PresentationExchange.createPresentationFromCredentials(listOf(sanctionsVcJwt), pd)
+
+      assertNotNull(presentationSubmission.id)
+      assertEquals(pd.id, presentationSubmission.definitionId)
+
+      assertEquals(1, presentationSubmission.descriptorMap.size)
+      assertNotNull(presentationSubmission.descriptorMap[0].id)
+      assertEquals("jwt_vc", presentationSubmission.descriptorMap[0].format)
+      assertEquals("$.verifiableCredential[0]", presentationSubmission.descriptorMap[0].path)
```

Comment:
What do you think about asserting against a presentation submission loaded from a json file? Would be nice for test-vectors, I think.

Reply:
I have a test vector PR tee'd up after this, so we can hav ea clean PR that shows how to add test vector tests